tests/MongoDB.Driver.Core.Tests/Core/Operations/IndexNameHelperTests.cs

```diff
@@ -52,5 +52,16 @@ public void GetIndexName_with_names_should_return_expected_result()
 
             result.Should().Be(expectedResult);
         }
+
+        [Fact]
+        public void GetIndexName_with_special_names_should_return_expected_result()
```

Comment:
Not sure that this test has a correct expected result. See comment in `GetIndexName`

Reply:
This test was renamed to `GetIndexName_with_wildcard_keys_should_return_expected_result`.
@rstam, @vincentkam could you please take a look at this test and the related method?
I'm not sure about this line:
`sb.Append("_1");`
should we update it somehow?

Reply:
I could not find this method.

Is there still something you need me to answer about this?

Reply:
no - after small clarification in the slack